### DISCUSSION TITLE
How to ignore parent style

### DISCUSSION THREAD

**Discussion by sonicsmooth** [AvaloniaUI/Avalonia#10592] [General]
I have a `TabControl` whose styles are passed into a nested `ColorPicker`.  I don't want this.  I want the regular `ColorPicker` styles.
How do I instantiate the `ColorPicker` and force it to use its default styles, or to ignore the parent styles?

I tried setting a few styles directly on the `ColorPicker`, but I'm not sure what values to use to "undo" the ones I changed in parent.

thanks

**Comment by maxkatz6** [comment]
You can't and you should do the opposite. Define your styles with a specific selectors. I.e. apply them on a specific Name of tab control or class names. Like Selector="TabControl.myTabControl" and then set Classes="myTabControl" on the tabcontrol.
Or, alternatively, you can use Control Themes.

**Reply by sonicsmooth** [reply]
Thanks for confirming my suspicion.  I was able to add a `Classes=options` on my tab control and on each `TabItem` and used `TabControl.options` and `TabItem.options` suffix on the selectors so the nested `ColorPicker`,  was not affected.

I was concerned that the `WrapPanel` would be a problem but it wasn't once I updated it to be like the example in https://docs.avaloniaui.net/docs/controls/tabcontrol.

thank you
